Dockerfile

```
@@ -94,7 +94,7 @@ USER root
 RUN --mount=target=/host \
     mkdir -p /local/rpms /local/migrations ./rpmbuild/RPMS \
     && ln -s /host/build/packages/*.rpm ./rpmbuild/RPMS \
-    && cp /host/build/packages/bottlerocket-${ARCH}-migrations-*.rpm /local/migrations \
+    && cp /host/build/packages/bottlerocket-${ARCH}-migrations-[0-9].[0-9]-[0-9].${ARCH}.rpm /local/migrations \
```

I think this is too specific.  It'd fail if we have a version component greater than 9, for example.  Here's another way that's pretty clear:

```suggestion
    && GLOBIGNORE="*debuginfo*" \
    && cp /host/build/packages/bottlerocket-${ARCH}-migrations-*.rpm /local/migrations \
    && unset GLOBIGNORE \
```

---

Found out that `*` doesn't match `/` in the path, so have to extend the pattern match with as many `/*` as there are subdirectories.

---

That's not true in my testing, and it's terribly brittle - I'd rather find another answer than have that many path components.  Are you sure it doesn't work with a single one?  Can we figure out what's different about our situations?

---

Something like this, that we worked out together offline:  `find /host/build/packages/ -maxdepth 1 -type f -name "bottlerocket-${ARCH}-migrations-*.rpm" -not -name '*debuginfo*' -exec cp {} /local/migrations`